etl/extract.py

```diff
@@ -0,0 +1,44 @@
+import re
```

Comment:
Looks like the contents of this file are helper functions, so maybe this could be `util.py`.